samtranslator/model/eventsources/push.py

```diff
@@ -129,7 +130,7 @@ def to_cloudformation(self, **kwargs):
         :rtype: list
         """
         function = kwargs.get('function')
-        
+
```

Comment:
sweeeeeeet!

Reply:
My editor (Atom) removes trailing whitespace automatically on save.

Reply:
That's cheating! I gave credit for your work, but turns out Atom did the work 😉 lol